src/loaders/OmeZarrLoader.ts

```diff
@@ -200,7 +200,13 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
     let channelCount = 0;
     for (const s of sources) {
       s.channelOffset = channelCount;
-      channelCount += s.omeroMetadata?.channels.length ?? s.scaleLevels[0].shape[s.axesTCZYX[1]];
```

Comment:
So just to check my understanding, this was failing before because `s.axesTCZYX[1]` evaluated to `-1`, which then caused the statement to effectively become:
```txt
channelCount += undefined ?? undefined
```

Is that right? and now in that failure case we instead increment by 1